libs/wingc/src/lsp/snapshots/completions/hide_private.snap

```diff
@@ -1,31 +1,5 @@
-  sortText: ab|node
-- label: onLift
-  kind: 2
-  detail: "preflight (host: IInflightHost, ops: Array<str>): void"
-  documentation:
-    kind: markdown
-    value: "```wing\npreflight onLift: preflight (host: IInflightHost, ops: Array<str>): void\n```\n---\nA hook called by the Wing compiler once for each inflight host that needs to use this resource inflight.\n### Parameters\n- `host` — `IInflightHost`\n- `ops` — `Array<str>`\n\n### Remarks\nYou can override this method to perform additional logic like granting\nIAM permissions to the host based on what methods are being called. But\nyou must call `super.bind(host, ops)` to ensure that the resource is\nactually bound."
-  sortText: ff|onLift
-  insertText: onLift($1)
-  insertTextFormat: 2
-  command:
-    title: triggerParameterHints
-    command: editor.action.triggerParameterHints
-- label: toString
-  kind: 2
-  detail: "preflight (): str"
-  documentation:
-    kind: markdown
-    value: "```wing\npreflight toString: preflight (): str\n```\n---\nReturns a string representation of this construct."
-  sortText: ff|toString
-  insertText: toString()
+[]
```

Comment:
This isn't a regression but this test isn't particularly useful anymore so it needs to be improved